src/chain/state.rs

```diff
@@ -70,7 +70,7 @@ impl State {
         &mut self,
         new_state: consensus::State,
     ) -> Result<(), StateError> {
-        // TODO(tarcieri): rewrite this using `Ord` impl on `consensus::State`
+        // TODO(tarcieri): rewrite this using `PartialOrd` impl on `consensus::State`
```

Comment:
Opened an upstream issue about how I think this should get fixed upstream in tendermint-rs to better reflect the Tendermint spec:

https://github.com/interchainio/tendermint-rs/issues/9